Automattic/wp-calypso
# Update/improve adding newsletter tier

This PR makes the process of adding a new newsletter tier nice in the paid newsletter importer flow. 

https://github.com/user-attachments/assets/312bca3b-2264-4583-8b4d-cdbb57d4a97a


## Proposed Changes

* Auto select the newly added newsletter tier. 
* Improvements the import subscriber button. 

## Why are these changes being made?
To improve the UX.

## Testing Instructions
* Go thought the new paid newsletter import flow. ( /import/newsletter/substack/example.com )
* Skip the content importing step. 
* On the subscriber step upload the CSV file. 
* Connect the Stripe account. 
* Add a newsletter tier. 
* Notice that the newly added newsletter tier is now selected. 

## Pre-merge Checklist
